app/models/group.rb

```diff
@@ -0,0 +1,39 @@
+class Group
+  attr_accessor :name
+  attr_accessor :description
+  attr_accessor :uuid
+  attr_writer   :users
```

Comment:
Because read method is explicitly defined.

Reply:
@bzwei @hsong-rh 
If groups are going to be defined in RBAC service why do we need one in app/models?

Reply:
@bzwei If these are helper functions they should live somewhere else maybe as a service.

Reply:
Should we be inheriting from this class and just adding the users method to it
https://github.com/mkanoor/rbac_api_client/blob/master/docs/GroupOut.md

Reply:
I don't want to inherit from `GroupOut` class because we don't care the operations such as parsing.
This is more like a DTO class while `GroupOut` is a DAO class.

Reply:
Strictly speaking I can leave the class with only attributes and keep it in the models folder and have a service class to do the conversion from `GroupOut` to `Group`.
Here I am taking a simpler approach to combine them and make it a class more like an AR but not actually backed by a database table.